lib/flipper/ui/views/layout.erb

```diff
@@ -16,7 +16,7 @@
     <![endif]-->
   </head>
   <body>
-    <div class="container">
+    <div class="container" style="max-width:600px;">
```

Comment:
Curious about the decision to go with the max width here. I assume the new layout just looks a lot better at a skinnier width compared to the old layout?

Reply:
Yep, I thought it looked much better shrunk up. It was just really long empty space and I couldn't come up with anything very valuable to fill it. It felt like a terrible way to do it so I'm open to a more standard way.

Reply:
I don't think there is anything wrong with setting a max width for the app if it looks good on all screens at that width. I generally try to stay away from inline styles. You could add another class to that `div` or set the `max-width` of the `container` class to `600px`. I don't know if that `container` class is used elsewhere though.

Reply:
Switched to a class in 011e63d